docs/resources/obs_bucket_policy.md

```diff
@@ -0,0 +1,43 @@
+---
+subcategory: "Object Storage Service (OBS)"
```

Comment:
S3 subcategory

Reply:
S3 subcategory is expected to be used for `s3_` resources only